src/scripts/popup.js

```diff
@@ -12,39 +12,44 @@ import { RandomUserGenerator } from './generators/RandomUserGenerator';
 import { UiNamesGenerator } from './generators/UiNamesGenerator';
 import { RandomProfileGenerator } from './generators/RandomProfileGenerator';
 
-$(document).ready(function () {
```

Comment:
Why is it changed from ready event?

Reply:
in [documentation](https://api.jquery.com/ready/) we can see the shorthand for **$( document ).ready()**, so I applied it.